api/transport_windows.go

```diff
@@ -0,0 +1,17 @@
+package api
```

Comment:
We might have discussed this previously, but my memory is foggy, is their a reason we only use npipe on windows and not for both?